bin/ch/ch.cpp

```diff
@@ -499,8 +504,29 @@ int _cdecl wmain(int argc, __in_ecount(argc) LPWSTR argv[])
     AssertMsg(lock, "failed to lock chakracore.dll");
 
     HostConfigFlags::HandleArgsFlag(argc, argv);
+#ifndef DEBUG
+    wchar_t fullPath[_MAX_PATH];
+    BSTR tmpPath;
+
+    if (_wfullpath(fullPath, argv[1], _MAX_PATH) == nullptr)
+    {
+      fwprintf(stderr, L"EFAIL: Failed to resolve full path\n");
+      return EXIT_FAILURE;
+    }
+
+    // canonicalize that path name to lower case for the profile storage
+    size_t len = wcslen(fullPath);
+    for (size_t i = 0; i < len; i++)
+    {
+      fullPath[i] = towlower(fullPath[i]);
+    }
 
+    tmpPath = SysAllocString(fullPath);
+    CComBSTR fileName(tmpPath);
```

Comment:
I *think* all that's needed in this `#ifndef` is `CComBSTR fileName(argv[1]);`, and the rest of work will be done in ExecuteTest to canonicalize the path.